backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -188,12 +195,17 @@ protected void createTablesLiquibaseMigration(
           System.getProperties().getProperty("liquibase.databaseChangeLogTableName");
 
       if (tableExists(con, config, changeLogTableName)) {
+        changeSetRemappingFile.ifPresent(
+            filename -> resetChangeSetLogData(jdbcCon, changeLogTableName, filename));
+
         String trimOperation;
         if (config.getRdbConfiguration().isMssql()) {
           LOGGER.info("MSSQL detected. Using custom update to liquibase filename records.");
           // looks like sql server doesn't support the "length" function, so hardcode it here.
-          trimOperation = "substring(FILENAME, 1, 19)";
+          trimOperation = "REPLACE(FILENAME, 'src/main/resources/', '')";
         } else {
+          // note: the leading slash here is a trick; the column doesn't have the leading slash, but
+          // adding it makes the length turn out right.
           trimOperation = "substring(FILENAME, length('/src/main/resources/'))";
```

Comment:
why does one have `/..../` and the other only has `.../`?

Reply:
see the comment directly above this line that I added.